src/mujincontrollerclient.cpp

```diff
@@ -15,6 +15,8 @@
 #include "controllerclientimpl.h"
 #include <boost/thread.hpp> // for sleep
 #include <boost/algorithm/string.hpp>
+#include <netdb.h>
+#include <arpa/inet.h>
```

Comment:
I think the implementation using `getaddrinfo` is fine, but should we consider an implementation using boost?

Reply:
I tried boost version here: https://github.com/mujin/controllerclientcpp/pull/176#issuecomment-2513333935

Boost seems easy to read. Should we go with boost (not sure how to decide tbh)? cc @yoshikikanemoto

Reply:
i don't see much difference. to me, `getaddrinfo` is easier to read or get info.